packages/model-viewer/src/test/three-components/ARRenderer-spec.ts

```diff
@@ -266,22 +265,7 @@ suite('ARRenderer', () => {
         expect(forward.y).to.be.equal(0);
         expect(cameraToHit.cross(forwardProjection)).to.be.closeTo(0, epsilon);
         expect(cameraToHit.dot(forwardProjection)).to.be.lessThan(0);
-      });
-
-      suite('after hit placement', () => {
-        let hitPosition: Vector3;
-
-        setup(async () => {
-          hitPosition = new Vector3(5, -1, 1);
-          await arRenderer.placeModel(hitPosition);
```

Comment:
I'm not thrilled with our testing here anyway, and I removed this method, so I just combined these tests.